Project/Project/Pages/Index.cshtml.cs

```diff
@@ -35,9 +35,24 @@ public IActionResult OnGet(string query)
             {
                 using (var webClient = new WebClient())
                 {
-                    string jsonString = webClient.DownloadString("https://pkgstore.datahub.io/core/country-list/data_json/data/8c458f2d15d9f2119654b29ede6e45b8/data_json.json");
+                    
+                    string countryString = webClient.DownloadString("https://pkgstore.datahub.io/core/country-list/data_json/data/8c458f2d15d9f2119654b29ede6e45b8/data_json.json");
+                    string result = "";
+
+                    
+
+                    try
```

Comment:
Implemented try catch block to handle exception

Reply:
Logging is a good idea, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute after the exception occurs, as if nothing went wrong.  That is worse than the default behavior: terminate the method when an exception occurs.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().